code/modules/tools/tool_extension.dm

```diff
@@ -50,6 +50,7 @@
 		return check_result
 
 	to_chat(user, SPAN_NOTICE("You begin [start_message || tool_archetype.use_message] \the [target] with \the [holder]."))
+	user.visible_message(SPAN_NOTICE("You begin [start_message || tool_archetype.use_message] \the [target] with \the [holder]."))
```

Comment:
A) The previous message becomes redundant.
B) Now it says "you" to everyone in the vicinity.